src/MeetingsSDKAdapter.js

```diff
@@ -210,12 +212,21 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     try {
       const sdkMeeting = this.fetchMeeting(ID);
       const {localAudio, localVideo} = this.meetings[ID];
-      const localStream = new MediaStream([localAudio.getAudioTracks()[0], localVideo.getVideoTracks()[0]]);
+      const [localStream] = await sdkMeeting.getMediaStreams(DEFAULT_MEDIA_SETTINGS);
 
       await sdkMeeting.join();
 
       // SDK requires to join the meeting before adding the local stream media to the meeting
       await sdkMeeting.addMedia({localStream, DEFAULT_MEDIA_SETTINGS});
```

Comment:
If both local audio and video are muted at this point, we could skip calling `addMedia`, that way we wouldn't have to immediately mute after joining. But, we would need some sort of way to actually add the media once the user unmutes. It would require utilizing the sdk meeting's state machine to determine if there is actually local media.

Reply:
Probably a future improvement, not required in this PR.

Reply:
that was my initial plan. But sdk `AddMedia` method does not behave properly when I unmute on a non-existent stream. that's why I just decided to mute the streams instead.